packages/manager/apps/pci-instances/src/components/cart/components/CartTotalPrice.component.tsx

```diff
@@ -0,0 +1,46 @@
+import { useCatalogPrice } from '@ovh-ux/manager-react-components';
+import { Text } from '@ovhcloud/ods-react';
+
+type TCartTotalPrice = {
+  price: number;
+  text?: string;
+  displayHourlyPrice?: boolean;
+  displayMonthlyPrice?: boolean;
```

Comment:
TODO (in generic implementation) : let the caller formats the price using a formatting function :)

Reply:
haha I was expecting this one! As we said, v.0 ;)